schism/mplink.c

```diff
@@ -222,6 +222,29 @@ int song_find_last_channel(void)
 
 // ------------------------------------------------------------------------
 
+int song_get_pattern_offset(int * n, song_note_t ** buf, int * r, int offset)
```

Comment:
I'd prefer the name `row` instead of `r` here. It's plenty short and much easier to read.

Reply:
fixing.